src/hooks/useKeyPress.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState, useCallback } from "react";
+
+export const useKeyPress = (targetKey: string) => {
+  const [keyPressed, setKeyPressed] = useState(false);
+
+  const downHandler = useCallback(
```

Comment:
you can create a single handler and perform this  `setKeyPressed(event.type === "keydown");`

Reply:
@Tushar-4781, updated the code based on your suggestion.